backend/src/core/BotProcess.js

```
@@ -1018,19 +1354,108 @@ process.on('message', async (message) => {
+                    if (!bot || !bot.pathfinder) return;
+                    const goal = new (require('mineflayer-pathfinder').goals.GoalBlock)(x, y, z);
+                    await bot.pathfinder.goto(goal);
+                },
+                attack: (entityId) => {
+                    if (!bot) return;
+                    const entity = bot.entities[entityId];
+                    if (entity) bot.attack(entity);
+                },
+                follow: (username) => {
+                    if (!bot || !bot.pathfinder) return;
+                    const player = bot.players[username];
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)